MAPL_Base/MAPL_CFIO.F90

```diff
@@ -9,7 +9,7 @@
 #include "unused_dummy.H"
 
 module MAPL_CFIOMod
-   use MAPL_ErrorHandlingMod
+   use MAPL_ExceptionHandling
```

Comment:
Not worth changing,   but ErrorReporting is probably more accurate than ExceptionHandling.   But the latter sounds more professional.